blocks/kanban-board/src/components/board/card/card.tsx

```diff
@@ -0,0 +1,61 @@
+import { useSortable } from "@dnd-kit/sortable";
+import { CSS } from "@dnd-kit/utilities";
+import clsx from "clsx";
+
+import { IconButton } from "../../icon-button/icon-button";
+import { DiscardIcon } from "../../icons/discard-icon";
+import { CardData, UpdateCardContentCallback } from "../types";
+import { EditableCardContent } from "./editable-card-content/editable-card-content";
+import styles from "./styles.module.scss";
+
+export const Card = ({
+  data,
+  onDelete,
+  updateCardContent,
+  readonly,
+}: {
+  data: CardData;
+  onDelete: () => void;
+  updateCardContent: UpdateCardContentCallback;
+  readonly?: boolean;
+}) => {
+  const {
+    attributes,
+    listeners,
+    setNodeRef,
+    transform,
+    transition,
+    isDragging,
+    isSorting,
+  } = useSortable({ id: data.id });
+
+  const style = {
+    transform: CSS.Transform.toString(transform),
+    transition,
+    opacity: isDragging ? 0.5 : 1,
+  };
+
+  return (
+    <div
+      className={clsx(
+        styles.wrapper,
+        (isDragging || isSorting) && styles.showHandCursor,
+      )}
+      ref={setNodeRef}
+      style={style}
+      {...attributes}
+      {...listeners}
+    >
+      <EditableCardContent
+        content={data.content}
+        onChange={(val) => updateCardContent(data.id, val)}
+        readonly={readonly}
+      />
+      {!readonly && (
+        <IconButton onClick={onDelete}>
+          <DiscardIcon />
+        </IconButton>
```

Comment:
I would move this to inside `EditableCardContent` so there is no need to repeat it in `StaticCard`

Reply:
https://github.com/hashintel/hash/pull/2234/commits/3afbe6abb587665457717a7cce0ad33f7e84d99c Moved it here, also renamed `EditableCardContent` -> `CardContent` since it also has button now.